docs/guides/transactions/running-background-jobs.md

```diff
@@ -122,47 +108,4 @@ class Program
 
 ## Exceptions in scheduled tasks
 
-Exceptions in scheduled tasks are logged to the [Administrator log](../working-with-starcounter/administrator-web-ui.md#log). For example, the following code will return "No exception":
-
-```csharp
-Handle.GET("/Hello", () =>
-{
-    try
-    {
-        Scheduling.RunTask(() => throw new Exception());
-        return "No exception";
-    }
-    catch
-    {
-        return "Exception";
-    }
-});
-```
-
-This is logged to the console:
-
-```text
-System.Exception: Exception of type 'System.Exception' was thrown.
-   at StarcounterApplication1.Program.<>c.<Main>b__0_1() in C:\Users\User\source\
-epos\StarcounterApplication1\StarcounterApplication1\Program.cs:line 15
-   at Starcounter.DbSession.<>c__DisplayClass5_0.<RunAsync>b__0() in C:\TeamCity\BuildAgent\work\sc-11226\Level1\src\Starcounter\DbSession.cs:line 216
-HResult=-2146233088
-```
-
-If you wait for the `Task`, the exception will be brought to the waiting thread and logged. For example, the following code will return "Exception" and the same exception as above will be logged:
-
-```csharp
-Handle.GET("/Hello", () =>
-{
-    try
-    {
-        Scheduling.RunTask(() => throw new Exception()).Wait();
-        return "No exception";
-    }
-    catch
-    {
-        return "Exception";
-    }
-});
-```
-
+Unhandled exceptions in scheduled tasks are logged to the [Administrator log](../working-with-starcounter/administrator-web-ui.md#log). Besides of this fact, exception handling is identical to that of a regular .Net task.
```

Comment:
I'm not sure I like how much this was shortened. People were confused on how exceptions worked with scheduling, that's why we added these examples.

Remember that people using Starcounter are often junior developers that don't have a good grasp of threading. 

I'm not sure though, we don't want to document how native .NET features work, and with the improved explanations above, these additional examples might not be needed. 

You decide whether to shorten it like this or not.

Reply:
I think people were confused with the old `Scheduling.ScheduleTask` call, when exception handling was unique (in a bad sense). What do you think about adding a link to some .net article on this topic?